### PR TITLE
Users model

### PR DESCRIPTION
**Pull request by fede1608**
# Summary
- User model migration
# Trello Card
[https://trello.com/c/DtTQ0iGd/26-modelo-de-usuario](https://trello.com/c/DtTQ0iGd/26-modelo-de-usuario)

### CODE REVIEW

File: migrations/migrations/20181023175544-users-model.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      },
+      password: {
+        type: Sequelize.STRING,
+        allowNull: false
+      },
+    });
+  },
+
+  down: (queryInterface, Sequelize) => {
+    /*
```
Comment (mpizzagalli) [inline migrations/migrations/20181023175544-users-model.js]: let's add some code here so this migration is rollbackable.

**Review by mpizzagalli — CHANGES_REQUESTED** [review]
You only added the migration to create the table, we should also add the model that binds to that table.

**Review by fede1608 — COMMENTED** [review]
Migration down code & model binding added

File: migrations/migrations/20181023175544-users-model.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      },
+      email: {
+        type: Sequelize.STRING,
+        allowNull: false,
+        unique: true
+      },
+      password: {
+        type: Sequelize.STRING,
+        allowNull: false
+      }
```
Comment (mpizzagalli) [inline migrations/migrations/20181023175544-users-model.js:29]: You also need to add some field for timestamps like `created_at` and `updated_at` or add to the model the option of `timestamps: false` so the orm knows that you don't have those columns in the table.

**Review by mpizzagalli — CHANGES_REQUESTED** [review]
Some little fix to avoid problems in the next card

File: migrations/migrations/20181023175544-users-model.js (hunk 2 above)
Comment (fede1608) [reply migrations/migrations/20181023175544-users-model.js:29]: I actually fixed that on the userCreate pr, should I add those fixes to this branch as well?

Comment (mpizzagalli) [reply migrations/migrations/20181023175544-users-model.js:29]: let's leave it there

**Review by mpizzagalli — APPROVED** [review]

**Pull request closed without merging**